.changeset/fix-workspace-protocol.md

```
@@ -0,0 +1,8 @@
+---
+"@pikku/cli": patch
+"@pikku/cloudflare": patch
+"pikku-vscode": patch
+"@pikku/kysely-mysql": patch
+---
+
+Replace workspace:* protocol with explicit npm version ranges in all package.json files. Fixes broken publishes where workspace:* was included literally in the npm registry.
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)